node-agent/workload_manager/src/workload_manager/workload_listener/error.rs

```diff
@@ -0,0 +1,27 @@
+use std::error::Error;
+use std::fmt;
+
+#[derive(Debug)]
+pub struct WorkloadListenerError {
+    details: String,
+}
```

Comment:
If you really need this, please use the `anyhow`crate instead. But afaict `WorkloadListenetError`is unused in this PR, so you should just not add it for now.

Reply:
This file was created because we wanted to have custom errors, however we figured out that we need to throw tonic::Status in our results. I'm deleting this now